pkgs/by-name/te/tetrio-desktop/tetrio-plus.nix

```diff
@@ -0,0 +1,120 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+, wasm-pack
+, rustc
+, rustPlatform
+, wasm-bindgen-cli
+, binaryen
+
+, yarn
+, fetchYarnDeps
+, fixup_yarn_lock
+, nodejs
+, asar
+
+, tetrio-src
+}:
+
+let
+  version = "unstable";
```

Comment:
Most packages seem to write this as `unstable-YYYY-MM-DD` with the date of the revision used?